README.md

```diff
@@ -1,51 +1,84 @@
-# Lalilo's Calendar Challenge
+# Lalilo Calendar Challenge
 
-As the confinement is taking away students from their teacher, teachers need to have a better insight on their student's activity. Natacha, our product owner, has been working hard to understand what teachers do need. It appears that a first step would be to display the student activity on a calendar view.
+As the confinement is taking away students from them, teachers need to have a better insight on their student's activity. Natacha, our product owner, has been working hard to understand what teachers do need. It appears that a first step would be to display the student activity on a calendar view.
 
 Marjorie, our designer, has produced an incredible calendar design.
 
-<img src="./calendar.png" width="200" />
+<img src="./assets/calendar.png" width="200" />
 
 Millions of teachers in the world are now dependent on you to have insights on their student's work. After working hard, Natacha and Marjorie left on holidays for 2 weeks. They trust you to handle this nicely.
 
-Your mission is to develop this calendar.
+Your mission is to develop this calendar. 
 
 ## How to get started
 
-The project can be started like this:
-
+- Make sure you have git, node, and npm or yarn installed locally
+- Clone this repo
+- Start the app : 
 ```bash
 yarn install && yarn start
 ```
-
-It has been tested on Node 12.16 but should work with most of Node versions.
-
-The tests can be launched with:
+- Launch test : 
 
 ```bash
 yarn test --watch
 ```
 
-## Specs
+## Guidelines
+
+- Solve the levels in ascending order. Split your work in at least one commit per level (always specifiy the level in the commit message)
+- Spend about 2h on the challenge, we don't expect you to finish (especially we don't expect junior to go beyond level 1 or 2) it but we do expect the code you produce is clean, maintainable and robust.
+- you're free to add any package that could help you (although we don't advise you to do so for the calendar, as Marjo has plenty of customization ideas)
+- Open a pull request to send your result
+
+### Level 1 :  Global activities overview
+
+<img src="./assets/level1.png" width="200" />
+
+Display the total number of active days :
+- you will find the data to display in <a href="./public/student-activity.json">the activity list</a> containing for your student : 
+  - the name of the exercise
+  - whether the exercise was done at home or at school
+  - the start and end timestamps for this exercise
+  - It should be fetched through an HTTP request as if it was served from an API.
```

Comment:
I did not get that part ? What do you expect the candidate to do ?

Reply:
It was something in Nico's spec that I kept. I guess that we should make an axios request like this axios.get(process.env(URL)/activities.json)